js/src/redux/actions/snippetEditor.js

```diff
@@ -1,4 +1,4 @@
-import { unescapeString } from "../../helpers/stringHelpers";
+import decodeHTML from "yoast-components/composites/OnboardingWizard/helpers/htmlDecoder";
```

Comment:
I think we should move the htmlDecoder to a more general location like `js/src/helpers`, because it is used in a non-onboardingwizard context.

Reply:
Changing the path in yoast components means we need to do a release of components. That's not worth it at this moment.